test/platform-stats.test.js

```diff
@@ -196,39 +196,6 @@ describe('platform-stats', () => {
       const { rows } = await pgClient.query('SELECT station_id, day::TEXT FROM daily_stations')
       assert.deepStrictEqual(rows, [{ station_id: VALID_STATION_ID, day: today }])
     })
-
-    it('updates top measurements participants yesterday materialized view', async () => {
```

Comment:
Basically just moved this test to a different describe block, since it no longer has to do with daily_stations.

Plus some other minor changes to work with new schema